sync.sh

```diff
@@ -2,9 +2,54 @@
 
 set -e
 
-# Read input arguments
-INPUT_DIR=$( cd -- "$1" > /dev/null && pwd )
-OUTPUT_DIR=$( cd -- "$2" > /dev/null && pwd )
+# Define Help function
+
+Help()
+{
+   # Display Help
+   echo "Tool to fill in belgian TST file from trade republic monthly report."
+   echo "This tool also creates a draft mail with the generated document and a qr code to pay the tax."
+   echo
+   echo "Syntax: sh sync.sh [-h|i|o|s|p|c]"
+   echo "options:"
+   echo "h     Display this help and exit."
+   echo "i     Set the input folder for the pdf(s), default value is 'input'."
+   echo "o     Set the output folder for the generated pdf(s) and qr code, default value is 'output'."
+   echo "s     Set the signature image file path, default value is 'data/signature.jpg'."
+   echo "p     Set the personal informations json file path, default value is 'data/citizen.json'."
+   echo "c     Set the Gmail credentials json file path, default value is 'data/credentials.json'."
+   echo
+}
+
+# Set default values for input arguments
+INPUT_DIR=$( cd -- "input" > /dev/null && pwd )
+OUTPUT_DIR=$( cd -- "output" > /dev/null && pwd )
```

Comment:
I am not too happy about that change, I would still like to provide the input and output folder as arguments of the script

Reply:
Ah, I just realized they can be provided as options now, can't you have both options and arguments?

Reply:
I think it would be redundant to be able to set them in multiple ways (and maybe confusing). I can just make them provided as argument if you prefer.